src/main/java/seedu/address/model/EventModelManager.java

```diff
@@ -0,0 +1,135 @@
+package seedu.address.model;
```

Comment:
Correct me if im wrong. but we are not using this class right?

Reply:
Not using now but might use if team wants more oop